src/sass/components/_tour.scss

```diff
@@ -37,14 +37,73 @@
 
 .popover-wrapper {
   position: absolute;
-  top: -10px;
+  left: -($spacing-xx-small);
+  top: -($spacing-small);
+
+  .epic-progress & {
+    top: -($spacing-medium);
+  }
 
   .logo-popover & {
-    top: unset;
+    top: $spacing-small;
+    left: $spacing-large;
+  }
+
+  .metecho-breadcrumb & {
+    left: $spacing-large;
+    top: -($var-spacing-x-small);
+  }
+
+  .project-placeholder & {
+    top: $spacing-medium;
+  }
+
+  .project-list & {
+    left: $spacing-none;
+    top: $spacing-xxx-small;
   }
 
   .page-title & {
-    top: 25px;
+    left: $spacing-large;
+    top: $var-spacing-medium;
+  }
+
+  .user-info & {
+    left: -($spacing-xx-small);
+    top: -($spacing-small);
+  }
+
+  .page-options & {
+    left: -($spacing-xx-small);
+    top: -($spacing-medium);
+  }
```

Comment:
@dvdherron same here, we could target `.page-title > .popover-wrapper` and then we probably won't need this. Although, I notice we are using a `-1rem` top value as opposed the the `.75rem` that other buttons use. Is there a reason for the `.25` difference?

Reply:
@stacyk I had originally thought it looked better to get all three of these on a similar level but I think the slight difference on narrow screens is actually fine. 

![image](https://user-images.githubusercontent.com/41588129/119972451-861c0800-bfb2-11eb-9173-45f516a44f57.png)

Reply:
me too @dvdherron